### PR TITLE
Corrected bitwidth graph dump for HAWQ

### PR DESCRIPTION
**Pull request by ljaljushkin**

### CODE REVIEW

File: nncf/quantization/algo.py [hunk 1]
```diff
@@ -1764,6 +1762,12 @@ def get_quantizer_setup_for_current_state(self) -> SingleConfigQuantizerSetup:
             retval.quantization_points[qp_id] = new_qp
         return retval
 
+    def apply_quantizer_setup_to_current_state(self, quantizer_setup: SingleConfigQuantizerSetup):
```
Comment (vshampor) [inline nncf/quantization/algo.py:1765]: But we have `apply_new_quantizer_setup` for that.

**Review by vshampor — CHANGES_REQUESTED** [review]

File: nncf/quantization/algo.py (hunk 1 above)
Comment (ljaljushkin) [reply nncf/quantization/algo.py:1765]: IMHO, `apply_new_quantizer_setup` is not  much for that. It creates builder, controller... why not just map?

Comment (vshampor) [reply nncf/quantization/algo.py:1765]: Your `apply_new_quantizer_setup_to_current_state` does not actually apply the setup, it merely sets up bitwidths while leaving the rest of the quantizer configuration as-is, which leads to a model ending up in an inconsistent state with respect to the quantizer configuration constraints.

Comment (ljaljushkin) [reply nncf/quantization/algo.py]: ok, makes sense. used `apply_new_quantizer_setup`

**Review by vshampor — APPROVED** [review]

**Pull request merged**